src/ocamlorg_package/lib/ocamlorg_package.mli

```diff
@@ -104,6 +104,10 @@ val license_file :
   kind:[< `Package | `Universe of string ] -> t -> string option Lwt.t
 (** Get the license of a package *)
 
+val changes_file :
+  kind:[< `Package | `Universe of string ] -> t -> (bool * string) option Lwt.t
```

Comment:
I'm not sure about returning a tuple with a bool that indicates whether this is a markdown file or not.

This is to be able to link to the correct file, but it could potentially be many formats. could we return the name of the file instead?

Also we should document what each of the tuple elements is.